### PR TITLE
Fix test with proxy

### PR DESCRIPTION
**Pull request by oraluben**
### What problem does this PR solve? 
#299 

### Check List 

Tests 

 - No code

Code changes

 - Has CI related scripts change

Side effects

Related changes

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,7 +19,7 @@ GOARCH := $(if $(GOARCH),$(GOARCH),"")
 GOENV  := GO15VENDOREXPERIMENT="1" CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH)
 CGOENV  := GO15VENDOREXPERIMENT="1" CGO_ENABLED=1 GOOS=$(GOOS) GOARCH=$(GOARCH)
 GO     := $(GOENV) go
-GOTEST := TEST_USE_EXISTING_CLUSTER=false go test
+GOTEST := TEST_USE_EXISTING_CLUSTER=false NO_PROXY=testhost go test
```
Comment (YangKeao) [inline Makefile]: Please add `testhost` to the end of `NO_PROXY` (split with `,`), but not override it

**Comment by you06** [comment]
The score is updated, check it out at https://pingcap.com/community-cn/tidb-usability-challenge/

**Pull request merged**